src/indicators/flash-message-container.js

```diff
@@ -47,7 +47,6 @@ function FlashMessageContainer({ messages, limit, onDismiss, ...rest }) {
         return (
           <FlashMessage
             key={message.id}
-            message={message}
```

Comment:
This was an accidental regression that I introduced in my work last year. I don't think it should be considered a breaking change because it's a bug and not documented (or supported by prop types)